lib/http_clients/creditas/person_deletion_api/person_deletion.ex

```diff
@@ -1,13 +1,30 @@
 defmodule HttpClients.Creditas.PersonDeletionApi.PersonDeletion do
   @moduledoc false
 
+  alias HttpClients.Creditas.PersonDeletionApi.Contact
+
   @type t :: %__MODULE__{
           id: String.t(),
           person_id: String.t(),
-          person_cpf: String.t()
+          person_cpf: String.t(),
+          contacts: List.t(Contact.t())
         }
 
   @derive Jason.Encoder
   @enforce_keys ~w(id person_id person_cpf)a
-  defstruct ~w(id person_id person_cpf)a
+  defstruct [:id, :person_id, :person_cpf, contacts: []]
+
+  @email_channels ["EMAIL"]
+  @spec person_emails(PersonDeletion.t()) :: List.t(String.t())
+  def person_emails(person_deletion), do: person_codes(person_deletion, @email_channels)
+
+  @phone_channels ["WHATSAPP", "PHONE", "SMS"]
+  @spec person_phones(PersonDeletion.t()) :: List.t(String.t())
+  def person_phones(person_deletion), do: person_codes(person_deletion, @phone_channels)
+
+  defp person_codes(person_deletion, channels) do
```

Comment:
```suggestion
  defp person_contact_values(person_deletion, channels) do
```
:nail_care: algo assim? achei estranho esse code até entender a struct